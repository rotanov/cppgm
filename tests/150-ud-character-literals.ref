invalid 'a'bcd
user-defined-literal 'c'_fasf _fasf character char 63
invalid 'fsafsa'fagasg
eof
