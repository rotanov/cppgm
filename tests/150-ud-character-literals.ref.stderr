ERROR: ud_suffix does not start with _: 'a'bcd
ERROR: ud_suffix does not start with _: 'fsafsa'fagasg
